locales/base/translation.json

```diff
@@ -77,7 +77,7 @@
     "title": "Your recovery phrase",
     "titleV1_90": "Your Recovery Phrase",
     "body": "Write these 12 words down somewhere safe, where it cannot be stolen or damaged.",
-    "bodyV1_90": "If you lose your recovery phrase, you will lose access to all funds held in {{appName}}.",
+    "bodyV1_90": "If you lose your recovery phrase, you will lose access to all of your assets on {{appName}}. Nobody, not even {{appName}} will be able to help you recover your account without your Recovery Phrase. Store it in a safe place and do not share it.",
```

Comment:
Is this a mento specific change? Wonder if it was design being out of date w/ actual implementation (applies to other copy changes)

Reply:
Also asked Kayla about this!

Reply:
I think this change is intended, as it matches the existing `postSetupBody` translation key.

Reply:
is this something that should've been done on Valora then?